### PR TITLE
Update OpenTelemetry story

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Relates to: https://github.com/kubewarden/kubewarden-controller/pull/304
Depends on: https://github.com/kubewarden/helm-charts/pull/136

Update OpenTelemetry story with the changes discovered in https://github.com/kubewarden/kubewarden-controller/pull/304.

Move to Helm charts for some components as their upstreams are in good shape now.
List Helm chart versions at time of writing for reproducibility.
Fix some values.yaml mistakes.

### CODE REVIEW

**Comment by viccuad** [comment]
Ready for review, but shouldn't be merged before https://github.com/kubewarden/helm-charts/pull/136.

File: docs/operator-manual/telemetry/metrics/01-quickstart.md [hunk 1]
```diff
@@ -72,7 +73,7 @@ helm install --wait --create-namespace \
 
 ## Install Kubewarden
 
-We can now install Kubewarden in the recommended way with the Helm chart.
+We can now install Kubewarden in the recommended way with the Helm charts.
```
Comment (divya-mohan0209) [inline docs/operator-manual/telemetry/metrics/01-quickstart.md:76]:
```suggestion
We can now install Kubewarden in the recommended way with Helm charts.
```

**Review by divya-mohan0209 — COMMENTED** [review]
Just one small nit. Rest LGTM!

**Comment by viccuad** [comment]
Many thanks @divya-mohan0209! merging then!

**Pull request merged**